meetings/2018-02-22.md

```diff
@@ -0,0 +1,184 @@
+# Node.js Foundation Community Committee Meeting 2018-02-22
+
+## Links
+
+* **Recording**: https://youtu.be/uZKB27_Xu4A
+* **GitHub Issue**: https://github.com/nodejs/community-committee/issues/245
+* **Minutes Google Doc**: https://docs.google.com/document/d/1ZeWyJLVu4-e-urEPGNW6MnjiGxeXFtbT7i6RC7CGiks/edit?usp=sharing
+
+## Present
+- Ben Michel  @obensource
+- Tierney Cyren  @bnb Maintainer
+- Olivia  @oe
+- Manil  @chowdhurian
+- Jem Bezooyen  @JemBijoux
+- Joe Sepi  @joesepi
+- Zibby Keaton @ZibbyKeaton
```

Comment:
I seem to be missing 
- Michael Dawson @nodejs/evangelism 

I was chairing so probably missed adding myself to the list